server/models/UserRequests.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+
+const userRequestsSchema = new mongoose.Schema({
+  user_id: {
+    type: Number,
+    required: true,
+    unique: true
+  },
+  sitter_id: {
+    type: Number,
+    required: true,
+    unique: true
+  },
+  start_date: {
+    type: Date,
+    default: Date.now(),
+  },
+  end_date: {
+    type: Date,
+    //7 Days later
+    default: () => Date.now() + 7*24*60*60*1000,
+  },
```

Comment:
I'm not sure that a `request` makes sense without start and end dates, so my suggestion would be to make these fields required and without any default values.